pages/accountLists.page.tsx

```diff
@@ -45,6 +45,16 @@ export const getServerSideProps = makeGetServerSideProps(async (session) => {
       query: GetAccountListsDocument,
     });
 
+    if (data.user.setup) {
+      // The user has not finished setting up, so start them on the tour
+      return {
+        redirect: {
+          destination: '/setup/start',
+          permanent: false,
+        },
+      };
+    }
```

Comment:
As the Tools part isn't built out yet, we should comment this out until we're ready to redirect people to the setup pages.

Reply:
I disabled the redirect functionality. Although this means that it can't be tested properly until tools go live. Another idea would be to do a feature-flag with an environment variable so the code runs in preview but not production. Let me know if you want further changes.

Reply:
I'm good with the feature-flag with an environment variable.

Reply:
I switched this to use a feature-flag. Set `DISABLE_SETUP_TOUR=true` to disable all setup tour logic.